Library/Homebrew/cmd/uninstall.rb

```diff
@@ -8,8 +8,11 @@
 require "cli/parser"
 require "cask/cmd"
 require "cask/cask_loader"
+require "uninstall"
 
 module Homebrew
+  extend Uninstall
```

Comment:
Instead: call `Uninstall.` methods explicitly (which make it a little easier to follow).

Reply:
Done